js/setup.js

```diff
@@ -0,0 +1,58 @@
+// Файл setup.js
+'use strict';
+var WIZARD = {
+  FIRST_NAMES: ['Иван', 'Хуан Себастьян', 'Мария', 'Кристоф', 'Виктор', 'Юлия', 'Люпита', 'Вашингтон'],
+  LAST_NAMES: ['да Марья', 'Верон', 'Мирабелла', 'Вальц', 'Онопко', 'Топольницкая', 'Нионго', 'Ирвинг'],
+  COAT_COLORS: ['rgb(101, 137, 164)', 'rgb(241, 43, 107)', 'rgb(146, 100, 161)', 'rgb(56, 159, 117)', 'rgb(215, 210, 55)', 'rgb(0, 0, 0)'],
+  EYES_COLORS: ['black', 'red', 'blue', 'yellow', 'green']
+};
+
+var WIZARDS_COUNT = 4;
+
+var userDialog = document.querySelector('.setup');
+userDialog.classList.remove('hidden');
+
+var similarListElement = userDialog.querySelector('.setup-similar-list');
+
+var similarWizardTemplate = document.querySelector('#similar-wizard-template')
+    .content
+    .querySelector('.setup-similar-item');
+
+var getRandomElement = function (array) {
+  return array[Math.floor(Math.random() * array.length)];
+};
+
+var generateWizards = function (wizardsData, wizardsCount) {
+  var wizards = [];
+
+  for (var i = 0; i < wizardsCount; i++) {
+    var wizard = {
+      name: getRandomElement(wizardsData.FIRST_NAMES) + ' ' + getRandomElement(wizardsData.LAST_NAMES),
+      coatColor: getRandomElement(wizardsData.COAT_COLORS),
+      eyesColor: getRandomElement(wizardsData.EYES_COLORS)
+    };
+
+    wizards.push(wizard);
```

Comment:
Можно не создавать отдельную переменную:
```suggestion
    wizards.push({
      name: getRandomElement(wizardsData.FIRST_NAMES) + ' ' + getRandomElement(wizardsData.LAST_NAMES),
      coatColor: getRandomElement(wizardsData.COAT_COLORS),
      eyesColor: getRandomElement(wizardsData.EYES_COLORS)
    });
```